main.go

```diff
@@ -176,6 +177,41 @@ func locateDotGit(dir string) (string, error) {
 	return "", fmt.Errorf("Couldn't find .git. Maybe you are not inside of a git repo")
 }
 
+func getURLAliases() (map[string]string, error) {
+	usr, err := user.Current()
+	if err != nil {
+		return map[string]string{}, err
+	}
+
+	path := path.Join(usr.HomeDir, ".gitconfig")
+
+	cfg, err := ini.Load(path)
+	if err != nil {
+		return map[string]string{}, err
+	}
+
+	sections := cfg.Sections()
+	aliases := map[string]string{}
+
+	for _, elem := range sections {
+		name := elem.Name()
+		isURLSection := strings.Contains(name, "url")
+		hasHostName := strings.Contains(name, "github") || strings.Contains(name, "gitlab")
+
+		if isURLSection && hasHostName {
+			section := cfg.Section(name)
+			key, err := section.GetKey("insteadOf")
+			if err != nil {
+				return map[string]string{}, err
+			}
+
+			aliases[key.Value()] = name[5 : len(name)-1]
```

Comment:
Why specifically 5? What are we even doing here? Are we parsing a URL? Is there any more reliable way to do that?

Reply:
My thought process was:

- I have `url "git@github.com:"` as section name.
- I want link string from inbetween quotation marks
- Just trim 5 chars (`url "`) from left and 1 (`"`) char from right

This solution was the simplest for me at that time.

Reply:
Not sure if I made changes the way you want. Is this what you meant?